lib/grinder_tools.dart

```diff
@@ -155,13 +160,15 @@ class Pub {
    * even if the pubspec.lock file is up-to-date with respect to the
    * pubspec.yaml file.
    */
-  static void get({bool force: false, String workingDirectory}) {
+  static String get({bool force: false, String workingDirectory}) {
```

Comment:
Not sure how useful it is to access the stdout of some of these commands.  If there are any that you would prefer to leave `void` just let me know which ones.  Would it be at all useful to instead return a `bool` here for whether `pub get` was actually run?